HHS/OPRE-OPS
# feat: ✨ Approve/decline a single agreement round 2

## What changed

- better layout for Review Cards
- move logix to react hooks module
- fixes render error on agreement meta data on review page and updates e2e tests
- adds batch api call for approving/declining change requests

Does not include

- confirmation modals or alerts

## Issue

#2349 

## How to test

1. goto agreement 9
2. make a budget change and send for approval
3. make a status change for the other budget line
4. review change requests and click link to approve page
5. should filter by action type and approve/decline the corresponding budget lines

## Screenshots

_If relevant, e.g. for a front-end feature_

## Definition of Done Checklist
